docs/upgrade.html

```diff
@@ -294,6 +327,23 @@ <h5><a id="upgrade_1010_streams" href="#upgrade_1010_streams">Upgrading a 0.10.0
     <li> Upgrading your Streams application from 0.10.0 to 0.10.1 does require a <a href="#upgrade_10_1">broker upgrade</a> because a Kafka Streams 0.10.1 application can only connect to 0.10.1 brokers. </li>
     <li> There are couple of API changes, that are not backward compatible (cf. <a href="/{{version}}/documentation/streams#streams_api_changes_0101">Streams API changes in 0.10.1</a> for more details).
          Thus, you need to update and recompile your code. Just swapping the Kafka Streams library jar file will not work and will break your application. </li>
+    <li> Upgrading from 0.10.0.x to 0.10.1.2 requires two rolling bounces with config <code>upgrade.from="0.10.0"</code> set for first upgrade phase
```

Comment:
Strictly speaking we cannot yet talk about 0.10.1.2 in 0.11.0.3 yet since we may not release 0.10.1.2 before releasing 0.11.0.3. I think it may not be of a big issue still leaving this here, but just try to point it out.

Reply:
You are right -- not sure how to resolve this... Maybe we can only fix this manually when an older bug fix release is done... It's a little annoying :(